### PR TITLE
domain-analysis

### PR DESCRIPTION
**Pull request by naili-m**
#### Descriptions:
New connector

### CODE REVIEW

File: domain-analysis/info.json [hunk 1]
```diff
@@ -0,0 +1,379 @@
+{
```
Comment (cs-pooja-singh) [inline domain-analysis/info.json:1]: can we remove the logo.xcf and medium.png?

File: domain-analysis/operations.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+FREQTABLE = os.path.dirname(os.path.realpath(__file__)) + '/freq/freqtable.freq'
+
+
+def analyze_domain(config, params):
+    try:
+        domain = params.get('domain')
+        fc = FreqCounter()
+        fc.load(FREQTABLE)
+        total_word_probability, average_probability = fc.probability(domain)
+        return {'Total Word Probability': total_word_probability, 'Average Probability': average_probability}
```
Comment (cs-pooja-singh) [inline domain-analysis/operations.py]: any specific reason the keys are space separate words, and not camelCased json keys?

File: domain-analysis/operations.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    except Exception as exp:
+        logger.exception('Error Executing Code: {}'.format(exp))
+        raise ConnectorError('Error Executing Code: {}'.format(exp))
+
+def get_domain_popularity(config, params):
+    try:
+        domain = params.get('domain')
+        list_date = params.get('list_date', None)
+        list_date = arrow.get(list_date).format("YYYY-MM-DD") if list_date else None
+        tranco = Tranco(cache=True, cache_dir=CACHE_DIR)
```
Comment (cs-pooja-singh) [inline domain-analysis/operations.py:46]: for how long does this file remain in /tmp? the top 1M list get refreshed daily. how does the connector action run on a subsequent day know that it has to refresh the file?

File: domain-analysis/info.json (hunk 1 above)
Comment (naili-m) [reply domain-analysis/info.json:1]: done

File: domain-analysis/operations.py (hunk 2 above)
Comment (naili-m) [reply domain-analysis/operations.py]: done

File: domain-analysis/operations.py (hunk 3 above)
Comment (naili-m) [reply domain-analysis/operations.py:46]: new function _purge_old_files() to clean up old files, was under the impression the tranco package does it.

**Review by cs-pooja-singh — APPROVED** [review]

**Pull request merged**